src/main/java/Duke.java

```diff
@@ -1,10 +1,123 @@
+import java.util.Iterator;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
+        initDuke();
+        String lineLogo = "-----------------------------";
+        Scanner in = new Scanner(System.in);
+        String line = " ";
+        String command = " ";
+        String commandSubject = " ";
+        String adverb = " ";
+        String timeline = " ";
+        ArrayList<Task> taskArrayList = new ArrayList<>();
+        Task currentTask;
+        String taskIncomplete = Character.toString((char) 0x274C);
+        String taskComplete = Character.toString((char) 0x2713);
+        int endPos = 0;
+        int slashPos = 0;
+
+        while(true){
+            line = in.nextLine();
+
+            if(line.equals("bye")){
+                break;
+            }
+
+            if(line.contains(" ")){
+                endPos = line.indexOf(" ");
+                command = line.substring(0, endPos);
+                commandSubject = line.substring(endPos + 1);
```

Comment:
Perhaps you could use different name to better differentiate command and commandSubject

Reply:
Replaced command with commandName, hope that makes it clearer